README.md

```diff
@@ -13,6 +13,7 @@ Deploys AWS CloudFormation Stacks.
   with:
     name: MyStack
     template: myStack.yaml
+    parameter-overrides: "MyParam1=myValue,MyParam2=${{ secrets.MY_SECRET_VALUE }}"
```

Comment:
What do you think of moving this to the description bellow? I think not everybody needs it, and many people do a copy & past.

Reply:
We could move it to the description below, but I personally find things easier to understand when they are used in an example, like in this pull request. I think users will know that if they don't have any Cloudformation stack parameters then they can just delete that line if they copy & paste. So I'd prefer to leave it how it is, but I'm not don't really mind. Another option would be to have a second example with all the values being used - then the user would have two examples - a minimal one with just the required fields and a full one with optional fields.

Reply:
Yeah, I prefer these too. You know what. I think we can move this forward as is.